README.md

```diff
@@ -88,14 +90,120 @@ If you have an existing (old zendev) environment and you wish to continue using
     * `cd ~/src`
 1. Initialize an environment, the metis name is arbitrary. This may take some time.
     * `zendev init metis`
+    * to use stable version of zendev (tags can be found in https://github.com/zenoss/product-assembly) 
+        * `zendev init -t <tag> <yourEnvironmentName>` (i.e.$  zendev init -t 7.0.18 zenoss718x)
+1. Go to the valid directory
+   * `cd ~/src`
 1. Use the previously created environment.
-    * `zendev use metis`
+    * `zendev use metis` or `zendev use <yourEnvironmentName>`
+1. Update docker - (https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+    * `sudo apt-get update`
+    * `sudo apt-get install docker-ce docker-ce-cli containerd.io`
 1. Build a development based zenoss docker image.
     * `zendev devimg`
+    * To build devimage with zenpacks:
+        * for RM: `zendev devimg -p resmgr`
+        * for CZ: `zendev devimg --clean --product cse`
+    * If you have docker errors: `sudo usermod -a -G docker $USER` and relogin
 1. Build Control-Center.
     * `cdz serviced; make clean build`
+    * if have an issue with CC building on 6.5+ or CZ:
+        * https://github.com/control-center/serviced#dev-environment
+        * `sudo sysctl -w vm.max_map_count=262144`
```

Comment:
I think we can replace it with some permanent variant:
`echo "vm.max_map_count = 262144" >> /etc/sysctl.conf && sysctl --system`